pkg/api/types.go

```diff
@@ -221,6 +224,17 @@ type GCEPersistentDisk struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDisk struct {
```

Comment:
Sure, but iSCSI and NFS could both be implemented by different kinds of backends in different environments.  while GCE PD acts like a block device it does not implement an iSCSI protocol. :)